docs/src/rsh/step/index.md

```diff
@@ -463,6 +472,13 @@ unknowable( Notter, Knower(var_0, ..., var_N), [msg] )
 A knowledge assertion that the participant `{!rsh} Notter` _does not_ know the results of the variables `{!rsh} var_0` through `{!rsh} var_N`, but that the participant `{!rsh} Knower` _does_ know those values.
 It accepts an optional bytes argument, which is included in any reported violation.
 
+```reach
+load: /examples/rps-7-loops/index.rsh
+range: 73-73
+```
+
+This example from the [Rock, Paper, Scissors!](##tut) sets both Alice's hand and salt as unknowable, which prevents Bob from cheating.
```

Comment:
It doesn't set them. It checks that they are unknowable.

Reply:
modified